.github/workflows/security-scan.yml

```
@@ -44,36 +43,27 @@ jobs:
         fetch-depth: 0
     
-    - name: Setup Node.js
-      uses: actions/setup-node@v4
+    - name: Setup Bun (pinned)
+      uses: oven-sh/setup-bun@v1
       with:
-        node-version: ${{ env.NODE_VERSION }}
-        cache: 'npm'
-        cache-dependency-path: 'package-lock.json'
+        bun-version: 1.1.34
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)